### PR TITLE
Table: convert Light DOM LESS to SCSS

### PR DESCRIPTION
**Pull request by BY00565233**
JIRA: SURF-1884

### LGTM's
- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

File: src/scss/components/table/_index.scss [hunk 1]
```diff
@@ -181,16 +181,18 @@
 
 // Use .hxTable.hxBound instead
 .hxTable--bordered {
-  &:extend(.hxTable.hxBound all);
+  @extend .hxBound;
+  @extend .hxTable;
```
Comment (CITguy) [inline src/scss/components/table/_index.scss]: This should be a single extend as `@extend .hxTable.hxBound;`

File: src/scss/components/table/_index.scss [hunk 2]
```diff
@@ -48,10 +48,10 @@
...
 
 // ===== States =====
-@state-selected-bg: @cyan-50;
+$state-selected-bg: $cyan-50;
 
 // ===== Tables =====
-@table-border-color: @gray-300;
+$table-border-color: $gray-300;
 
 /* ========== Tables ========== */
```
Comment (100stacks) [inline src/scss/components/table/_index.scss]: Please updated all single line comments to use double slash (`//`) syntax:

```sass
// ========== Tables ==========
```

File: src/scss/components/table/_index.scss [hunk 3]
```diff
@@ -48,10 +48,10 @@
...
```
Comment (100stacks) [inline src/scss/components/table/_index.scss]: Please convert layout comment [Line 58-68] to use triple slash (`///`) syntax for documentation.

File: src/scss/components/table/_index.scss (hunk 2 above)
Comment (BY00565233) [reply src/scss/components/table/_index.scss]: Done the changes.

File: src/scss/components/table/_index.scss (hunk 3 above)
Comment (BY00565233) [reply src/scss/components/table/_index.scss]: Done the changes.

File: src/scss/components/_index.scss [hunk 4]
```diff
@@ -14,6 +14,6 @@
 @import "pill/index";
 @import "reveal/index";
 @import "selector-strip/index";
-@import "toast/index";
```
Comment (CITguy) [inline src/scss/components/_index.scss]: Please add back Toast

File: src/less/components.less [hunk 5]
```diff
@@ -36,7 +36,9 @@
 @import 'components/popover/index';
 @import 'components/radio/index';
 @import 'components/search/index';
-@import 'components/table/index';
+@import 'components/tabset/index';
+@import 'components/text-input/index';
+@import 'components/textarea/index';
```
Comment (CITguy) [inline src/less/components.less]: Revert the addition of these lines. You've got Tabset, Text Input, and Textarea imported twice (once in Lines 39-41, another in Lines 42-44)

**Pull request merged**